.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: "Version to use for the release."
+        required: true
+        default: "X.Y.Z"
+      developmentVersion:
+        description: "Version to use for further development"
+        required: true
+        default: "X.Y.Z-RIE"
+jobs:
+  Release:
+    environment: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Configure Git User # may be removed after https://github.com/actions/checkout/issues/13 is resolved
+        run: |
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+          git config user.name "${{ github.actor }}"
+      - name: Set up python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      - name: Build
+        run: make compile-with-docker-all
```

Comment:
It would be great if we can run the tests too. In theory the tests already ran before when the PR was merged, but it wouldn't hurt to run the tests before releasing too (and of course not release if the tests fail).

Reply:
Done!